Resources/public/js/apps/plugins/ezsearch-routeappplugin.js

```diff
@@ -0,0 +1,31 @@
+YUI.add('ezsearch-routeappplugin', function (Y) {
+    Y.namespace('eZSearch.Plugin');
+
+    Y.eZSearch.Plugin.RouteAppPlugin = Y.Base.create('ezsearchRouteAppPlugin', Y.Plugin.Base, [], {
+        initializer: function () {
+            var app = this.get('host');
+
+            app.views.ezsearchSearchView = {
+                type: Y.View,
+            };
+
+            app.route({
+                name: "viewSearch",
+                path: "/ezsearch/search",
+                view: "ezserachSearchView",
```

Comment:
typo here, and given the same typo is not repeated when declaring the view, this probably does not work very well :)